model/Individual/Classes/Element.md

```diff
@@ -0,0 +1,33 @@
+SPDX-License-Identifier: Community-Spec-1.0
```

Comment:
The convention is that you don't create a new class file (as there is not really a new class).

You put the `External properties restrictions` info on the top-level (namespace) file.